packages/client/hmi-client/src/components/workflow/tera-simulate-ensemble-node-ciemss.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div>
+	<div v-if="!disableRunButton">
```

Comment:
Is there a reason this is on div and not button?
Im not sure why this div exists? Im not certain

Reply:
do you have something against possibly unnecessary divs?

Reply:
If we are cleaning things up in the area it seems to make sense to remove them
Its not a hill I will die on though

Reply:
I have something against unnecessary div, as it get's hard to understand what's happening.